src/main/java/io/aiven/kafka/connect/transforms/ExtractTimestamp.java

```diff
@@ -91,13 +99,36 @@ public R apply(final R record) {
         );
     }
 
-    public static final class Value<R extends ConnectRecord<R>> extends ExtractTimestamp<R> {
-        // There's an implementation only for value, not for key.
-        // We provide $Value class anyway for the consistency sake
-        // and in case we need a $Key version in the future as well.
-    }
-
     @Override
     public void close() {
     }
+
+    protected abstract String dataPlace();
```

Comment:
I'm a bit confused with the naming `dataPlace`. Is it possible rename it to `timestampField` or `timestampPropertyName` or something else?

Reply:
It's the convention from other classes. "Place" being `key` or `value`, not the name of the field.
I admit it's strange :D, but better ideas for it are welcome

Reply:
Well, `keyOrValue` might be one